src/components/FileRepo/CustomColumns/ActionsColumn.js

```diff
@@ -68,7 +76,14 @@ const ActionItems = ({ value, fence, hasAccess, theme }) => {
             <ControlledIcon fill={theme.lightBlue} />
           </Tooltip>
         )}
-      </Column>
+      </ButtonWrapper>
+      <ButtonWrapper>
+        {hasAccess && (
+          <CavaticaOpenModalWrapper fileIds={[value]}>
+            <CavaticaLogo fill={theme.lightBlue} width={16} />
+          </CavaticaOpenModalWrapper>
```

Comment:
Do we wanna show something else if `!hasAccess`? Or maybe should we just move the whole `hasAccess` check wrap both the fence thing and cavatica?

Reply:
The design in this case just has nothing. Ask Kim?